@uportal/waffle-menu/src/WaffleMenu.js

```diff
@@ -124,13 +124,17 @@ class WaffleMenu extends Component {
     url: PropTypes.string,
     debug: PropTypes.bool,
     buttonColor: PropTypes.string,
+    defaultIcon: PropTypes.string,
+    truncateLength: PropTypes.string,
   };
 
   static defaultProps = {
     url: '/uPortal/api/v4-3/dlm/portletRegistry.json',
     oidcUrl: '/uPortal/api/v5-1/userinfo',
     buttonColor: '#fff',
     debug: false,
+    defaultIcon: '/ResourceServingWebapp/rs/tango/0.8.90/32x32/categories/applications-other.png',
```

Comment:
ESLint is yelling about this
https://travis-ci.org/uPortal-contrib/uPortal-web-components/builds/418933766#L519

See if running `npm run format` at the top level of this project clears it up.
If not `// eslint-disable-next-line max-len` will.